testdata/p4_16_samples/pna-action-selector-1.p4

```diff
@@ -15,7 +15,7 @@ limitations under the License.
 */
 
 #include <core.p4>
-#include "pna.p4"
+#include "dpdk/pna.p4"
```

Comment:
I think, generic testcases shouldn't be modified with dpdk specific header.

Reply:
done